python/phylanx/ast/physl.py

```diff
@@ -399,6 +404,9 @@ def _compile_or_load(self):
             self.__src__ = self._generate_physl(self.ir)
             self.__ast__ = phylanx.ast.generate_ast(self.__src__)
 
+            if self.doc_src is not None:
+                self.__src__ = self.doc_src
```

Comment:
Why would we need to replace the `__src__` here if we have successfully loaded it from the database?